File: JavaScript/프로그래머스/0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.js
function solution(n) {
    var answer = 0;
    var i=1;
    var flag=0;
    while(true){
        if(7*i>=n){
            flag=i;
            break;
        }
        else {
            i++;
        }
    }
    answer=flag;
    return answer;
}

File: 프로그래머스/0/120814. 피자 나눠 먹기 （1）/피자 나눠 먹기 （1）.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

int solution(int n) {
    int answer = 0;
    int seven = n/7;
    if (n%7==0)
        answer=seven;
    else {
        answer=seven+1;
    }
    return answer;
}